modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/dhs.scala

```diff
@@ -0,0 +1,8 @@
+package edu.gemini.seqexec.model
+
+object dhs {
+
+  // Common data types
+  type ObsId = String
```

Comment:
Is this a placeholder?  The observation ID should eventually be ported to Scala, placed in `core` and cross compiled once squants is ready.